flash/core/classification.py

```diff
@@ -22,14 +22,22 @@
 
 class ClassificationPostprocess(Postprocess):
 
+    def __init__(self, multi_label: bool = False, save_path: Optional[str] = None):
+        super().__init__(save_path=save_path)
+        self.multi_label = multi_label
+
     def per_sample_transform(self, samples: Any) -> Any:
-        return torch.argmax(samples, -1).tolist()
+        if self.multi_label:
+            return F.sigmoid(samples).tolist()
```

Comment:
Should we be hardcoding using sigmoid?

Reply:
Yes, we need to write it somewhere in the pipeline.